hack/run-test-e2e.sh

```diff
@@ -86,11 +37,26 @@ export E2E_TOPOLOGY_MANAGER_SCOPE="${E2E_TOPOLOGY_MANAGER_SCOPE}"
 echo "TM config policy=[${E2E_TOPOLOGY_MANAGER_POLICY}] scope=[${E2E_TOPOLOGY_MANAGER_SCOPE}]"
 ${BIN_DIR}/e2e-nrop-rte.test ${NO_COLOR} --ginkgo.v --ginkgo.timeout=5h --ginkgo.fail-fast --ginkgo.flake-attempts=2 --ginkgo.junit-report=${REPORT_DIR}/e2e.xml --ginkgo.skip='\[Disruptive\]|\[StateDirectories\]|\[NodeRefresh\]|\[local\]' --ginkgo.focus='\[release\]'
 
-if [ $? -ne 0 ]; then
-    echo "Failed to run NRO RTE tests"
-    exit 1
-fi
-
 if [ "$ENABLE_SCHED_TESTS" = true ]; then
-  test_sched
+  # Run install test suite
+  echo "Running NROScheduler install test suite"
+  ${BIN_DIR}/e2e-nrop-sched-install.test ${NO_COLOR} --ginkgo.v --ginkgo.timeout=5h --ginkgo.fail-fast --ginkgo.junit-report=${REPORT_DIR}/sched-install.xml
+
+  echo "Running Functional Tests: ${GINKGO_SUITS}"
+  # -v: print out the text and location for each spec before running it and flush output to stdout in realtime
+  # -timeout: exit the suite after the specified time
+  # -r: run suites recursively
+  # --fail-fast: ginkgo will stop the suite right after the first spec failure
+  # --flake-attempts: rerun the test if it fails
+  # -requireSuite: fail if tests are not executed because of missing suite
+  ${BIN_DIR}/e2e-nrop-sched.test ${NO_COLOR} --ginkgo.v --ginkgo.timeout=5h --ginkgo.fail-fast --ginkgo.flake-attempts=2 --ginkgo.junit-report=${REPORT_DIR}/e2e-sched.xml
+
+  echo "Running NROScheduler uninstall test suite";
```

Comment:
If it used only for CI purposes then the uninstall part just consumes time, because the cluster would get deleted anyway.

Reply:
you're right. I'm reluctant to remove it just yet because if memory serves we do some test in the deinstall path. I can soublecheck and remove if I'm remember wrongly.

Reply:
No tests on uninstall path, but let's discuss the removal in a separate PR

Reply:
ack